# Bioinformatics tools for pathogen genomics

I am involved in the development of several tools for pathogen genomic surveillance, outbreak investigation and metagenomics detection. 

They can be found in the following repositories:

 - https://github.com/INSaFLU - **INSaFLU-TELEVIR platform (online tool available at: https://insaflu.insa.pt/)**, which is a free bioinformatics web-based (but also locally installable) suite that deals with primary sequencing data (Illumina, Ion Torrent and Oxford Nanopore Technologies reads) towards: i) **metagenomics virus detection** (from reads to virus detection); ii) **routine genomic surveillance** (from reads to mutations detection, consensus generation, virus classification, alignments, “genotype-phenotype” screening, phylogenetics, integrative Nextstrain phylogeographical and temporal analysis etc). Role: Scientific coordinator.


- https://github.com/insapathogenomics - Repository of our Genomics and Bioinformatics Unit of the National Institute of Health Doutor Ricardo Jorge (INSA), Portugal

  Among other tools, I am coordinating the development of:
  
  https://github.com/insapathogenomics/ReporTree - **ReporTree: a surveillance-oriented tool to strengthen the linkage between pathogen genetic clusters and epidemiological data**
  
  https://github.com/insapathogenomics/algn2pheno - **algn2pheno: a bioinformatics tool for rapid screening of genetic features (nt or aa changes) potentially linked to specific phenotypes**

  https://github.com/INSaFLU/findONTime - **findONTime: A bioinformatics tool for real-time metagenomics virus detection analysis using ONT technology and the INSaFLU-TELEVIR platform**
